src/app.js

```diff
@@ -58,6 +59,8 @@ export const start = async args => {
     process.exit(0);
   } else if (args[0] === '--reset-config') {
     conf.all = defaultConfig;
+  } else if (args[0] === '--count' && args[1]) {
+    conf.set('refCount', parseInt(args[1]));
```

Comment:
This is creating a side effect of the flag option persisting. Is that the intention? I think options being passed should just act as an overriding flag for a single execution. If we need to persist options through the cli instead of editing the config file, then we can create a ticket to check for a persist/save option in the future.

Reply:
yes it was intentional, I'll modify it for a single execution then.

Reply:
I agree that this should be for single execution only.

Reply:
We will need to add error handling as well for cases where the end user uses a `flag` but does not pass a value or passes an invalid value like... `--count one`....etc..

Reply:
@jwu910 @kienD  modified it.